src/components/Header/Header.jsx

```diff
@@ -3,6 +3,7 @@ import { useNavigate } from "react-router-dom";
 import "./header.css";
 import { useAuth } from "../../utils/useAuth";
 import { useEffect, forwardRef } from "react";
+import { MenuOutlined } from "@ant-design/icons";
```

Comment:
Does the hamburger dropdown work? I can't really figure out how to get it to work in my firefox browser. Maybe just send me a video when you get a chance of it working on your system

Reply:
https://github.com/rknizzle/propermesh-view/assets/111093132/d07e9b07-ecdc-4fdf-ad52-b8aa7ff71b51

Reply:
There might be something weird about it, i'll look into later. It was working fine last night

Reply:
Ok sick! I think I just don't really know how to use the phone emulator in my browser lol

Reply:
I think I'd prefer the options to be all or nothing. So like either all the options on the header or all of them in the burger menu rather than being able to have some on the header and some in the menu.

Super low priority though we can definitely merge this in before doing that. I just documented it in this issue: https://github.com/rknizzle/propermesh-view/issues/52

Reply:
roger that!